raccoongang/edx-platform
# fix 500 error

**Description:** There has been a 500 error on the MAMA servers http://prntscr.com/ov6zut
if press "Download course user list" with empty field "Course ID or dir": http://prntscr.com/ov6zee
Need to show message "Не правильно заполнено поле Course ID or dir". here: http://prntscr.com/ov70d8

**Youtrack:** https://youtrack.raccoongang.com/issue/SKILLONOMY-131

**Testing instructions:**

1. Log in as staff
2. Enter sysadmin page
3. Go to courses page
4. Click "download course users" with blank or wrong value
5. Message "Не правильно заполнено поле Course ID or dir" must appear

**Post merge:**
